### PR TITLE
Add config for network

### PR DESCRIPTION
**Pull request by KedziaPawel**
* add network config to environments variable
* fix tests

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
-module.exports = {
-  sentryDsn: 'https://dc67fd6d5c2949f2a6853e60eb69d899@o429172.ingest.sentry.io/5375289',
-  network: bitcoin.alt_networks.bitcoinvaultTestnet,
+const { SENTRY_DSN, HOST, PORT, BTCV_NETWORK } = Config;
+
+export default {
+  sentryDsn: SENTRY_DSN || '',
+  host: HOST || '188.166.204.85',
+  port: PORT || '50001',
+  network: bitcoin.alt_networks[BTCV_NETWORK || 'bitcoinvault'],
```
Comment (Kamill90) [inline config/index.js:11]: when we have proper .env you can remove this config file.

Comment (KedziaPawel) [reply config/index.js:11]: do you mean the default values or the whole file?

Comment (Kamill90) [reply config/index.js:11]: file

Comment (KedziaPawel) [reply config/index.js:11]: we cannot delete the config file for couple of reasons:
1. We need some level of abstraction for react-native-config library so it won't spread in the whole app
2. In the config in the future we can have some properties that are not in the env
3. We have some properties like a network that is not taken directly from env but computed based on env

**Pull request merged**